packages/generator-particle-base/src/generators/app/index.ts

```diff
@@ -82,7 +93,14 @@ module.exports = class extends Generator {
         options: configOptions[results.config],
       }
     }
-
+    fs.writeFileSync(
```

Comment:
Can we move this fs call outside of the constructor. Feels cleaner to have a method called `writeParticleJson()`.